---
layout: publications
modified: 2020-07-15
comments: false
---

<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

<script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>

<div class="category-box">
<a href="#2019"><span class="category-item">2019</span></a><a href="#2017"><span class="category-item">2017</span></a><a href="#2016"><span class="category-item">2016</span></a>
</div>

<a name="2019">&nbsp;</a>
<h1>2019</h1>

<div class="pub">
<div class='altmetric-embed' data-badge-type='donut' data-doi="10.7554/eLife.46497"></div>
<div class="__dimensions_badge_embed__" data-doi="10.7554/eLife.46497" data-hide-zero-citations="true" data-legend="hover-bottom" data-style="small_circle"></div>
    <h3><a href=" https://doi.org/10.7554/eLife.46497" target="_new">Single-amino acid variants reveal evolutionary processes that shape the biogeography of a global SAR11 subclade</a></h3>
    <span class="pub-authors">Delmont TO<sup>☯</sup>, <span class="pub-member-author">Kiefl E<sup>☯</sup></span>, Kilinc O, Esen ÖC, Uysal I, Rappé MS, Giovannoni S, Eren AM</span>
    <span class="pub-co-first-authors"><sup>☯</sup>Co-first authors</span>
    <div class="pub-info">
    <div class="pub-featured-image">
    <a href="../images/pubs/delmond_and_kiefl_sar11_saavs.jpg"><img src="../images/pubs/delmond_and_kiefl_sar11_saavs.jpg" style="max-width: 100px; max-height: 80px; width: auto; border: none; height: auto; margin: 0 auto; display: block; transform: translateY(15%);"/></a>
    </div>
    <div class="pub-highlights">
    <span style="display: inline-block; padding-bottom: 5px;">- Introduces <a href="http://merenlab.org/2015/07/20/analyzing-variability/#an-intro-to-single-nucleotidecodonamino-acid-variation" target="_blank">'single-amino acid variants'</a> (SAAVs) and demonstrates <b>the use of SAAVs to tease apart evolutionary processes that shape the biogeography and genomic heterogeneity within a SAR11 population</b> through metagenomics.</span><br><span style="display: inline-block; padding-bottom: 5px;">-  A first attempt to link population genetics and the predicted protein structures to explore <i>in silico</i> <b>the intersection beetween protein biochemistry and evolutionary processes</b> acting on an environmental microbe.</span><br><span style="display: inline-block; padding-bottom: 5px;">-  An application of <a href="https://peerj.com/articles/4320/" target="_blank">metapangenomics</a> to define <b>subclades of SAR11 based on gene content and ecology</b>.</span><br><span style="display: inline-block; padding-bottom: 5px;">-  Reproducible bioinformatics workflow is <a href="http://merenlab.org/data/sar11-saavs/" target="_blank">here</a>. <a href="https://doi.org/10.7554/eLife.46497.040" target="_blank">Reviewer criticism</a> and <a href="https://doi.org/10.7554/eLife.46497.041" target="_blank">our responses</a> are also available.</span>
    </div>
    </div>
    <span class="pub-journal"><b>eLife</b>, 8:e46497.</span>
</div>


<a name="2017">&nbsp;</a>
<h1>2017</h1>

<div class="pub">
<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1021/acs.jpcc.7b08691"></div>
<div class="__dimensions_badge_embed__" data-doi="10.1021/acs.jpcc.7b08691" data-hide-zero-citations="true" data-legend="hover-bottom" data-style="small_circle"></div>
    <h3><a href=" https://doi.org/10.1021/acs.jpcc.7b08691" target="_new">Evaluation of surface-enhanced Raman spectroscopy substrates from single-molecule statistics</a></h3>
    <span class="pub-authors"><span class="pub-member-author">Kiefl E</span>, Kiefl RF, dos Santos DP, Brolo AG</span>
    <div class="pub-info">
    <div class="pub-featured-image">
    <a href="../images/pubs/kiefl_SERS.png"><img src="../images/pubs/kiefl_SERS.png" style="max-width: 100px; max-height: 80px; width: auto; border: none; height: auto; margin: 0 auto; display: block; transform: translateY(15%);"/></a>
    </div>
    <div class="pub-highlights">
    <span style="display: inline-block; padding-bottom: 5px;">- Demonstrates power-law statistics of surface-ehanced Raman spectroscopy (SERS) hotspots can be used to <b>assess the quality of SERS substrates</b>.</span><br><span style="display: inline-block; padding-bottom: 5px;">-  Extends the theory of truncated Pareto-distributed single-molecule SERS statistics to multi-hotspot substrates. </span>
    </div>
    </div>
    <span class="pub-journal"><b>The Journal of Physical Chemistry C</b>, 121(45):25487-25493.</span>
</div>


<a name="2016">&nbsp;</a>
<h1>2016</h1>

<div class="pub">
<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1021/acsnano.6b01817"></div>
<div class="__dimensions_badge_embed__" data-doi="10.1021/acsnano.6b01817" data-hide-zero-citations="true" data-legend="hover-bottom" data-style="small_circle"></div>
    <h3><a href=" https://doi.org/10.1021/acsnano.6b01817" target="_new">Robust Magnetic Properties of a Sublimable Single-Molecule Magnet</a></h3>
    <span class="pub-authors"><span class="pub-member-author">Kiefl E</span>, Mannini M, Bernot K, Yi X, Amato A, Leviant T, Magnani A, Prokscha T, Suter A, Sessoli R, Salman Z</span>
    <div class="pub-info">
    <div class="pub-featured-image">
    <a href="../images/pubs/kiefl_SMM.png"><img src="../images/pubs/kiefl_SMM.png" style="max-width: 100px; max-height: 80px; width: auto; border: none; height: auto; margin: 0 auto; display: block; transform: translateY(15%);"/></a>
    </div>
    <div class="pub-highlights">
    <span style="display: inline-block; padding-bottom: 5px;">- Demonstrates an equivalence in the magnetic properties between bulk and nanofilm configurations of a single-molecule magnet (SMM) using <b>muon spin spectroscopy</b></span><br><span style="display: inline-block; padding-bottom: 5px;">-  Discovers a rare instance in which a single molecule magnet maintains its chemical structure and magnetic properties when sublimated into nanofilm, <b>an important precursor for using SMMs for information storage</b>.</span>
    </div>
    </div>
    <span class="pub-journal"><b>ACS Nano</b>, 10(6):5663-5669.</span>
</div>

<div class="pub">
<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1016/j.bpj.2016.10.039"></div>
<div class="__dimensions_badge_embed__" data-doi="10.1016/j.bpj.2016.10.039" data-hide-zero-citations="true" data-legend="hover-bottom" data-style="small_circle"></div>
    <h3><a href=" https://doi.org/10.1016/j.bpj.2016.10.039" target="_new">Intact telopeptides enhance interactions between collagens</a></h3>
    <span class="pub-authors">Shayegan M, Altindal T, <span class="pub-member-author">Kiefl E</span>, Forde NR</span>
    <div class="pub-info">
    <div class="pub-featured-image">
    <a href="../images/pubs/shayegan_collagen.jpg"><img src="../images/pubs/shayegan_collagen.jpg" style="max-width: 100px; max-height: 80px; width: auto; border: none; height: auto; margin: 0 auto; display: block; transform: translateY(15%);"/></a>
    </div>
    <div class="pub-highlights">
    <span style="display: inline-block; padding-bottom: 5px;">- Uses <b>optical tweezers-based microrheology</b> to quantify the <b>viscoelasticity</b> of triple-helical collagen molecules, with and without non-helical flanking regions called telopeptides, which are known to be <b>critical for self-assembly</b>.</span><br><span style="display: inline-block; padding-bottom: 5px;">-  This work suggests that <b>telopeptides facilitate transient intermolecular interactions between collagen proteins</b></span>
    </div>
    </div>
    <span class="pub-journal"><b>Biophysical journal</b>, 111(11):2404-2416.</span>
</div>


